dhis-2/dhis-api/src/main/java/org/hisp/dhis/appmanager/AppManager.java

```diff
@@ -36,13 +36,49 @@
 import java.util.Collection;
 import java.util.List;
 
+import com.google.common.collect.ImmutableSet;
+
 /**
  * @author Saptarshi Purkayastha
  */
 public interface AppManager
 {
     String ID = AppManager.class.getName();
 
+    ImmutableSet<String> BUNDLED_APPS = ImmutableSet.of(
+        // Javascript apps
+        "app-management",
+        "cache-cleaner",
+        "capture",
+        "dashboard",
+        "data-administration",
+        "data-visualizer",
+        "data-quality",
+        "datastore",
+        "event-reports",
+        "event-visualizer",
+        "import-export",
+        "interpretation",
+        "maintenance",
+        "maps",
+        "menu-management",
+        "messaging",
+        "pivot",
+        "reports",
+        "scheduler",
+        "settings",
+        "tracker-capture",
+        "translations",
+        "usage-analytics",
+        "user",
+        "user-profile",
+        
+        // Struts apps
+        "approval",
+        "dataentry",
+        "maintenance-mobile"
+    );
```

Comment:
Ideally this wouldn't duplicate the app cloning list in `dhis-web-apps`, but I don't think it would be trivial to refactor them at the moment.  Eventually this should go away when we move to the `/apps/xyz` app base

Reply:
Agreed.